uftrace.c

```diff
@@ -558,6 +561,16 @@ static int parse_option(struct opts *opts, int key, char *arg)
 		opts->hide = opt_add_string(opts->hide, arg);
 		break;
 
+	case 'O':
+		pos = strchr(arg, '@');
```

Comment:
Do you want me to implement '@hide' suffix with trigger instead of this?

Reply:
Yeah, I think that's the way to go.  But we could have this first and then convert it later..